besu/terraform/aws/ibftScale/files/orion/orion.yml

```diff
@@ -0,0 +1,22 @@
+---
+- hosts: localhost
+  connection: local
+  force_handlers: True
+
+
+  roles:
+   
+    - role: pegasyseng.orion
+      vars:
+        orion_version: PARAM_ORION_VERSION
+        orion_download_url: "PARAM_ORION_DOWNLOAD_URL"
```

Comment:
why is this one in quotes?

Reply:
That is what ansible needs ...
I'm not 100% sure, but that is what the download urls are in other places. AND it works :-)

Reply:
it's what YAML needs, really... if there's a colon anywhere in the string, it needs to be quoted.